bucoffea/config/monojet.yaml

```diff
@@ -42,8 +42,8 @@ default:
       pt: 18
       eta: 2.3
     branch:
-      id: Tau_idDecayMode
-      iso: Tau_idMVAoldDM2017v2
+      id: Tau_idDecayModeNewDMs
+      iso: Tau_idDeepTau2017v2p1VSjet
```

Comment:
Lets only change VBF for now pls

Reply:
Sure, will revert monojet config back to original